pkg/tso/allocator_manager.go

```diff
@@ -315,64 +314,6 @@ func (am *AllocatorManager) GetMember() ElectionMember {
 	return am.member
 }
 
-// SetLocalTSOConfig receives the zone label of this PD server and write it into etcd as dc-location
-// to make the whole cluster know the DC-level topology for later Local TSO Allocator campaign.
-func (am *AllocatorManager) SetLocalTSOConfig(dcLocation string) error {
-	serverName := am.member.Name()
-	serverID := am.member.ID()
-	if err := am.checkDCLocationUpperLimit(dcLocation); err != nil {
-		log.Error("check dc-location upper limit failed",
-			logutil.CondUint32("keyspace-group-id", am.kgID, am.kgID != 0),
-			zap.Int("upper-limit", int(math.Pow(2, MaxSuffixBits))-1),
-			zap.String("dc-location", dcLocation),
-			zap.String("server-name", serverName),
-			zap.Uint64("server-id", serverID),
-			errs.ZapError(err))
-		return err
-	}
-	// The key-value pair in etcd will be like: serverID -> dcLocation
-	dcLocationKey := am.member.GetDCLocationPath(serverID)
-	resp, err := kv.
-		NewSlowLogTxn(am.member.Client()).
-		Then(clientv3.OpPut(dcLocationKey, dcLocation)).
-		Commit()
-	if err != nil {
-		return errs.ErrEtcdTxnInternal.Wrap(err).GenWithStackByCause()
-	}
-	if !resp.Succeeded {
-		log.Warn("write dc-location configuration into etcd failed",
-			logutil.CondUint32("keyspace-group-id", am.kgID, am.kgID > 0),
-			zap.String("dc-location", dcLocation),
-			zap.String("server-name", serverName),
-			zap.Uint64("server-id", serverID))
-		return errs.ErrEtcdTxnConflict.FastGenByArgs()
-	}
-	log.Info("write dc-location configuration into etcd",
-		logutil.CondUint32("keyspace-group-id", am.kgID, am.kgID > 0),
-		zap.String("dc-location", dcLocation),
-		zap.String("server-name", serverName),
-		zap.Uint64("server-id", serverID))
-	go am.ClusterDCLocationChecker()
-	return nil
-}
-
-func (am *AllocatorManager) checkDCLocationUpperLimit(dcLocation string) error {
-	clusterDCLocations, err := am.GetClusterDCLocationsFromEtcd()
-	if err != nil {
-		return err
-	}
-	// It's ok to add a new PD to the old dc-location.
-	if _, ok := clusterDCLocations[dcLocation]; ok {
-		return nil
-	}
-	// Check whether the dc-location number meets the upper limit 2**(LogicalBits-1)-1,
-	// which includes 1 global and 2**(LogicalBits-1) local
-	if len(clusterDCLocations) == int(math.Pow(2, MaxSuffixBits))-1 {
-		return errs.ErrSetLocalTSOConfig.FastGenByArgs("the number of dc-location meets the upper limit")
-	}
-	return nil
-}
-
 // GetClusterDCLocationsFromEtcd fetches dcLocation topology from etcd
 func (am *AllocatorManager) GetClusterDCLocationsFromEtcd() (clusterDCLocations map[string][]uint64, err error) {
```

Comment:
How about this one?

Reply:
This PR does not delete all the functions; it is already large enough. We just need to ensure that the deleted content does not affect normal operation. The remaining functions will be removed in subsequent PRs.